src/mlpack/methods/linear_svm/linear_svm.hpp

```diff
@@ -264,6 +270,26 @@ class LinearSVM
   bool fitIntercept;
 };
 
+/**
+ * Callback test function, based on the EndOptimization callback function.
+ */
+class CallbackTestFunction
+{
+ public:
+    CallbackTestFunction() : calledEndOptimization(false) {}
+
+    template<typename OptimizerType, typename FunctionType, typename MatType>
+    void EndOptimization(OptimizerType& /* optimizer */,
+                       FunctionType& /* function */,
+                       MatType& /* coordinates */)
+    {
+        calledEndOptimization = true;
+    }
+
+    //! Track to check if callback is executed
```

Comment:
Sorry another picky style comment, there is a missing stop at the end.

Reply:
Done.